packages/vue-client/src/components/boards/ParallelGoBoard.vue

```diff
@@ -10,7 +10,13 @@ function forEachMoveModifyStone(
 ) {
   Object.keys(moves).forEach((player_str) => {
     const player = Number(player_str);
-    const move = Coordinate.fromSgfRepr(moves[player]);
+    const move_str = moves[player];
+    if (move_str.length !== 2) {
+      // special moves like "pass" and "resign"
+      return;
+    }
+    const move = Coordinate.fromSgfRepr(move_str);
```

Comment:
This makes me wonder if this is actually a bug in `fromSgfRepr`.

Current behavior:

- output { x: 15, y: 0 }

Potentially improved behavior:

- output {x: -1, y: -1}
- Throw an error ("pass" is not how passes are represented in SGF anyway)